server/schema/ddl.sql

```diff
@@ -433,4 +433,4 @@ CREATE CONSTRAINT TRIGGER domainmetadata_trigger AFTER INSERT OR UPDATE OR DELET
 ON domainmetadata
 DEFERRABLE INITIALLY DEFERRED
 FOR EACH ROW
-EXECUTE PROCEDURE domainmetadata_update();
\ No newline at end of file
+EXECUTE PROCEDURE domainmetadata_update();
```

Comment:
You need to bump the schema version [here](https://github.com/atomia/atomiadns/blob/797868293e6ba4d2cb1b0ff4c44e32b117f866fc/server/schema/ddl.sql#L43)

Reply:
Resolved.